src/plugins/console/public/application/containers/editor/legacy/console_editor/editor.tsx

```diff
@@ -198,6 +199,26 @@ function EditorUI({ initialTextValue, setEditorInstance }: EditorProps) {
       }
     }
 
+    function restoreFolds() {
+      if (editor) {
+        const foldRanges = storage.get('folds', []);
```

Comment:
Nit: It would be great to store storage keywords in the `StorageKeys` enum, as it is done at https://github.com/elastic/kibana/blob/19e84d96aca45a52958f4396e420105620b63c66/src/plugins/console/public/services/storage.ts#L13

WDYT about it?